api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/EligibilityResponseTestDataFactory.java

```diff
@@ -14,4 +15,11 @@ public static EligibilityResponse anEligibilityResponse() {
                 .build();
     }
 
+    public static EligibilityResponse aNonMatchingEligibilityResponse() {
```

Comment:
Can we refactor this an the other method in this Test Data Factory to use a common base method returning the builder so you can just change the eligibilityStatus?

Reply:
Done.